lib/components/checkboxes/attributes.js

```diff
@@ -4,6 +4,7 @@ import Classlist from 'classlist';
 import queryAll from '../../commons/query-all';
 import isSelected from '../../commons/is-selected';
 import getLabel from '../../commons/get-label';
+import SELECTOR from './selector';
```

Comment:
Why not do:

```js
export const selector = '.some-selector.for-something'
```

?

Reply:
seems awkward to have to `import { selector } from './attributes';` in the other files that consume this selector...whats the harm in having the selector in its own file?

Reply:
_updated pr to not have selector.js files_